src/fileActions.jsx

```diff
@@ -206,8 +206,8 @@ const RenameItemModal = ({ path, selected }) => {
             : path.join("/") + "/" + name;
 
         cockpit.spawn(is_current_dir
-            ? ["mv", path.join("/"), newPath]
-            : ["mv", path.join("/") + "/" + selected.name, newPath],
+            ? ["mv", "-T", path.join("/"), newPath]
```

Comment:
This added line is not executed by any test.

Reply:
Hmm this should be covered. Let's try to cover that in a follow up but let's think about it:

This is about renaming the current dir, which we no longer support, so this is dead code.

Reply:
https://github.com/cockpit-project/cockpit-files/issues/405